prediction/src/tests/test_identify_trained_model_predict.py

```diff
@@ -0,0 +1,57 @@
+import os
+
+import numpy as np
+import pytest
+
+from ..algorithms.identify import trained_model
+
+
+def skip_slow_test():
```

Comment:
Sorry for giving feedback at such a late state but you're not supposed to re-implement this function as it was already added to [prediction/src/tests/test_endpoints.py](https://github.com/concept-to-clinic/concept-to-clinic/blob/757dfea9959db8e79f9b4ab5c04748aea711ca77/prediction/src/tests/test_endpoints.py) :) Are you able to import it from there?

Reply:
nice catch @WGierke!